[Failing Test]: Samza XVR Test Failure on release-2.47.0 branch

### What happened?

The Samza cross-language validates runner test suite is failing on the release branch with the following error in the TestDebeziumIO_BasicRead test:

```
2023/04/12 17:19:07  (): java.lang.UnsupportedOperationException: The ActiveBundle does not have a registered bundle checkpoint handler.
2023/04/12 17:19:07 Job state: FAILED
    ptest.go:108: Failed to execute job: job go0testdebeziumio0basicread0521-jenkins-0412171850-7f7d8f6d_49001b62-7368-42c6-8be8-92ce416f7f7c failed
--- FAIL: TestDebeziumIO_BasicRead (44.53s)
```

### Issue Failure

Failure: Test is continually failing

### Issue Priority

Priority: 0 (outage / failing precommit test impacting development)

### Issue Components

- [ ] Component: Python SDK
- [X] Component: Java SDK
- [X] Component: Go SDK
- [ ] Component: Typescript SDK
- [X] Component: IO connector
- [ ] Component: Beam examples
- [ ] Component: Beam playground
- [ ] Component: Beam katas
- [ ] Component: Website
- [ ] Component: Spark Runner
- [ ] Component: Flink Runner
- [X] Component: Samza Runner
- [ ] Component: Twister2 Runner
- [ ] Component: Hazelcast Jet Runner
- [ ] Component: Google Cloud Dataflow Runner

--- a/sdks/go/test/integration/integration.go
+++ b/sdks/go/test/integration/integration.go
@@ -187,6 +187,8 @@ var samzaFilters = []string{
 	"TestMapStateClear",
 	"TestSetState",
 	"TestSetStateClear",
+	// TODO(https://github.com/apache/beam/issues/26126): Java runner issue (AcitveBundle has no regsitered handler)
+	"TestDebeziumIO_BasicRead",
 }
 
 var sparkFilters = []string{

